src/index.ts

```diff
@@ -1,47 +1,15 @@
-import dotenv from "dotenv";
-import "module-alias/register";
-import schedule from "node-schedule";
-import FixturesFactory from "./services/fixtures/factory";
-import PreMatch from "./services/thread/pre-match";
+import dotenv from 'dotenv';
+import express from 'express';
+import routing from '@milanbot/routing';
```

Comment:
```suggestion
import { scheduleRequestHandler, preMatchRequestHandler, errorHandler } from '@milanbot/routing';
```
When importing named exports, it is best to export them like this and be explicit about it.

Reply:
Ok!